src/Footer.js

```diff
@@ -0,0 +1,13 @@
+const Footer = () => {
+    return (
+        <footer>
+        <p>Created at  
+            <span className="secondColor"><a href="#"> Juno College </a></span> 
+            by 
+            <span className="firstColor"><a href="#"> Gabriella Penev</a></span>
```

Comment:
The footer css is so cool! I would suggest adding your portfolio in the href link